shims/pom.xml

```diff
@@ -42,7 +42,6 @@
                 <activeByDefault>true</activeByDefault>
             </activation>
             <modules>
-                <module>spark301emr</module>
```

Comment:
Does nothing build spark301emr now?  I'd rather rip out the code if we aren't even going to regularly verify it remains buildable.

Reply:
no, nothing builds it. I figured if we don't have their jars no reason to build it.  I left it thinking it might be useful for EMR folks.  I can rip it out or we can talk to them to get more details on how they build?

Reply:
IMO either we build it or we remove it.  If we leave it unbuilt, I guarantee it will rot over time to the point it no longer builds.  If it's useful to the EMR team then we should have them pull in the changes to their repo where they build the EMR shim.

Reply:
ok, I'll just remove it

Reply:
We might want to do that as a separate PR rather than as part of adding Databricks.  This is already a really big PR.

Reply:
ok